### ISSUE TITLE
Using cert files

### ISSUE THREAD

**Issue by simonberger** [aacotroneo/laravel-saml2#95]
I was curious how I can use the possibility to load the certs from file as described in the config.

```
// Usually x509cert and privateKey of the SP are provided by files placed at
// the certs folder. But we can also provide them with the following parameters
```

Looking at the code it seemed that isn't really possible. At least not "Laravel-Like".
If you would define ONELOGIN_CUSTOMPATH to for example config/ we could create a certs folder in config. If that's no option for you or won't work (didn't test yet) the comment could be more clear stating the vendor-package certs folder is meant.

**Comment by aacotroneo** [comment]
that's the default settings from onelogin, with roughly the default comments, and we just overwrite some configs. isn't it finding you custom paths to your certs? i think it should (code will be in onelogin dependency).
if that works, I'd rather not laraveli-ze it so we don't get more coupled to the library than needed.

**Comment by simonberger** [comment]
If I set the constant it should work, sure.
I just thought it fits in the boot call of your package.

I understand if you have a fifferent opinion, but a more clear comment would be good then.

**Comment by aacotroneo** [comment]
oh I see..  I haven't understood well - we can change the comment to avoid confusion

**Comment by peetersdiet** [comment]
@simonberger It's not been discussed yet with @aacotroneo, but maybe PR #104 can mean something to you?

**Comment by aacotroneo** [comment]
sorry, missed that PR - I'll review later today!

**Comment by peetersdiet** [comment]
Thanks! Curious to hear your feedback, but no rush. 👍

**Comment by alexcaglairi** [comment]
@aacotroneo is it possible to contact you in pvt? can you give your email? Thanks

**Comment by aacotroneo** [comment]
@alexcaglairi yes.. write to cotale@hotmail.com ... let me know when you read this message so I can remove it to avoid massive spam!

**Comment by alexcaglairi** [comment]
Sorry, i'm here :) i'm ready :)

**Comment by alexcaglairi** [comment]
@aacotroneo can you answer please?
